README.md

```diff
@@ -207,7 +220,45 @@ service:
     curl -XPOST localhost:9080/reindexing/fedora/path -H"Content-Type: application/json" \
         -d '["broker:queue:solr.reindex","broker:queue:fixity","broker:queue:triplestore.reindex"]'
 
-## Troubleshooting
+#### Properties
+| Name      | Description| Default Value |
+| :---      | :---| :----   |
+| reindexing.enabled | Enables/disables the reindexing component. Enabled by default | true | 
+| reindexing.error.maxRedeliveries | Maximum redelivery attempts | 10 | 
+| reindexing.stream | Reindexing jms message stream | broker:queue:reindexing | 
+| reindexing.rest.host | Reindexing service host | localhost | 
+| reindexing.rest.port | Reindexing service port | 9080 |
+| reindexing.rest.prefix | Reindexing rest URI prefix | /reindexing | 
+
+### ActiveMQ Service
+
+This implements a connector to an ActiveMQ broker.
```

Comment:
Is this the service that is used to listen for message from fedora? Should we further specify that?